.github/workflows/node-unit-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [16.x]
```

Comment:
this is a bit extreme for my taste, node 14 is supported up to apr 2023 in general, getting rid of it way before seems bad from a support point of view.

Reply:
if AbortController is 15.4 and up, we can remove 14 and keep 16 only. We do odd numbers just when they are the latest

_Originally posted by @asturur in https://github.com/fabricjs/fabric.js/pull/7827#discussion_r835652818_

Reply:
yep i found my comments later.

Reply:
https://www.npmjs.com/package/abortcontroller-polyfill

Look fine.
And polyfills `fetch` as well

Reply:
I'm leaving tests on v16 because they use AbortController